### DISCUSSION TITLE
save row does not trigger Postgresql

### DISCUSSION THREAD

**Discussion by feurer98** [Budibase/budibase#8651] [Help]
I have two tables where I always copy the uuid of the entry when creating a record.

When testing via pgAdmin, the records are copied. Data entered through Budibase triggers, but the uuid is not copied.
Is it an INSERT operation or an UPDATE?

**Comment by melohagan** [comment]
Hey @feurer98 

It may be that the UUID datatype is not yet supported for Postgres in Budibase. Could you please share the schema (CREATE script) for your Postgres table please, and I can try to reproduce.
